src/internal/components/ThemeEditor.tsx

```diff
@@ -71,13 +77,18 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
 
     // use layout from localStorage when in dev mode
     if (templateMetadata.isDevMode && !!localHistoryArray) {
-      const localHistories = JSON.parse(localHistoryArray) as History[];
+      const localHistories = JSON.parse(
+        localHistoryArray
+      ) as History<AppState>[];
       const localHistoryIndex = localHistories.length - 1;
       if (localHistoryIndex >= 0) {
         devLogger.log("Theme Dev Mode - Using layout data from local storage");
         setPuckInitialHistory({
           // @ts-expect-error https://github.com/measuredco/puck/issues/673
-          histories: localHistories,
+          histories: localHistories.map((h) => {
+            // strip ui state
```

Comment:
Would we want to do this everywhere and not just devMode specifically?

Reply:
For Theme Manager, the object from content in prod mode doesn't have ui data on it
For Layout Editor, we could do this but I'm not sure we would want to... I think it makes sense that if you hide a sidebar and refresh the page it would still be hidden